### PR TITLE
support for stats

### PR DESCRIPTION
**Pull request by iloveitaly**
### All submissions

* [x] I have followed the [Contributing Guidelines](https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md)
* [x] I have checked to ensure there aren't other open [Pull Requests](https://github.com/lra/mackup/pulls) for the same update/change

### Adding/updating Application X Support

* [x] This PR is only for one application
* [x] It has been added to the list of supported applications in the [README](https://github.com/lra/mackup/blob/master/README.md)
* [x] Changes have been added to the WIP section of the [CHANGELOG](https://github.com/lra/mackup/blob/master/CHANGELOG.md)
* [x] Syncing does not break the application
* [x] Syncing does not compete with any syncing functionality internal to the application
* [x] The configuration syncs the minimal set of data
* [x] No file specific to the local workstation is synced
* [x] No sensitive data is synced

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -626,6 +626,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [Startupizer2](http://appledoc.gentlebytes.com/startupizer/)
 - [Stata](http://www.stata.com/)
 - [Stay](https://cordlessdog.com/stay/)
+- [Stats](https://github.com/exelban/stats)
```
Comment (lra) [inline README.md:629]: Can you keep this list AZ ordered?

Comment (iloveitaly) [reply README.md:628]: @Ira done!

**Review by lra — APPROVED** [review]
Thanks

**Comment by chrisgrieser** [comment]
Hi, sorry to bother you via this channel, but could you maybe make a release? Support for Stats has been merged ~4 months ago, and yet the latest release of mackup still does not include support for it 🙏

**Pull request merged**